src/functionalTest/java/uk/gov/hmcts/probate/functional/documents/SolBaCcdServiceDocumentsTests.java

```diff
@@ -2150,6 +2151,14 @@ public void verifyGenerateSolsCoverSheetAdmonWill() {
 
     }
 
+    @Test
+    public void verifyGenerateSolsCoverSheetAuthenticatedTranslationOfWill() {
+        String payload = "/caseprogress/04f-caseCreated.json";
+        String response = getDocumentTextAtPath(payload, VALIDATE_PROBATE_URL, "solsCoversheetDocument");
+        assertTrue(response.contains(AUTHENTICATED_TRANSLATION_WILL_TEXT));
```

Comment:
I think I would prefer this AUTHENTICATED_TRANSLATION_WILL_TEXT value to be hard-coded in the test itself, because if the value changes for some reason, we want the test to highlight it... same for others below too